### DISCUSSION TITLE
在开源的世界里如何应对搭便车者？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#67] [Announcements]
# 在开源的世界里如何应对搭便车者？ - 「开源之道」

https://opensourceway.community/posts/the_way_of_open_source/how-to-repose-freerider-in-open-source/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

先不论如何分析/鉴别/应对/治理/... 搭便车客/ freerider ;

尼采的断言很靠谱, 很多屠龙少最后也变成了新的一条龙...

另外坊间有断言: `开源软件吞噬ed世界`

这其实说的就是 GPL 创造的 `惠左性`/传染性, 想在开源世界当一个纯粹的 搭便车客/ freerider 其实很难:
因为在 FLOSS 中, 并没有很清晰的 upstream/downstream 边界;

无论多么专业的 搭便车客/ freerider, 
嘦开始使用 开源产品, 
就不得不面对必然的 尝试/配置/理解/改进/定制/...
就算是自我封闭不向上反馈任何自己的经验,
也已经在 搭便车客/ freerider 组织部内部制造出一批开源用户了...

所以, 不必上心去 识别/对抗 搭便车客/ freerider 们, 
还是继续完善 FLOSS/自由|开源社区 内部文化,
提升同化率, 就象中国文化融合其它历史民族文化类似...
